File: client_manage_web/src/main/webapp/js/service/sellersService.js
//服务层
app.service('sellersService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../sellers/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../sellers/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../sellers/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../sellers/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../sellers/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../sellers/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../sellers/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});

File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/SellersService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbSellers;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SellersService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbSellers> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbSellers sellers);


	/**
	 * 修改
	 */
	public void update(TbSellers sellers);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbSellers findOne(String username);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String[] usernames);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbSellers sellers, int pageNum, int pageSize);
	
}
